versions.txt

```diff
@@ -1,2 +1,4 @@
 # List of packages to bump:
 # Format: [package_name]@[new_version]
+
+acid@3.0.0
```

Comment:
Based on the comment from Lee earlier in the PR (not going to pretend I actually understand these changes) should there be a minor version bump for alloy here?

Reply:
Good catch, yes we will also need that since we've added a new API to `Slider`.